utils/incentive.js

```diff
@@ -124,41 +124,36 @@ const incentiveCompleted = async (req, res) => {
 }
 
 const eligibleForIncentive = async (req, res) => {
-    logIPAddress(req);
-    setHeaders(res);
-
-    if(req.method === 'OPTIONS') return res.status(200).json({code: 200});
-    if(req.method !== 'GET') return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+     logIPAddress(req);
+     setHeaders(res);
 
-    const authorized = await APIAuthorization(req);
+     if (req.method === 'OPTIONS') return res.status(200).json({code: 200});
+     if (req.method !== 'GET') return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+     if (!req.query.round) return res.status(400).json(getResponseJSON('Round query parameter missing!', 400));
+     if (!['baseline', 'followup1', 'followup2', 'followup3'].includes(req.query.round)) return res.status(400).json(getResponseJSON('Invalid round!', 400)); 
+     if (req.query.limit && parseInt(req.query.limit) > 1000) return res.status(400).json(getResponseJSON('Bad request, the limit cannot exceed more than 1000 records!', 400));
```

Comment:
Should we use 1000 as fallback value and proceed the request?

Reply:
I'm going to say no.

1) This is fairly standard behavior for our codebase at the moment
2) Keeping it like this will force sites to update their code to conform to the standards we set.